Sources/Core/AWSSDKIdentity/Sources/AWSSDKIdentity/AWSCredentialIdentityResolvers/ProfileAWSCredentialIdentityResolver.swift

```diff
@@ -205,7 +230,8 @@ public struct ProfileAWSCredentialIdentityResolver: AWSCredentialIdentityResolve
                 return try await STSWebIdentityAWSCredentialIdentityResolver(
                     configFilePath: configFilePath,
                     credentialsFilePath: credentialsFilePath,
-                    profileName: profile.name
+                    profileName: profile.name,
+                    credentialFeatureIDs: credentialFeatureIDs
```

Comment:
In the case of STSWebIdentity and SSO resolvers, `credentialFeatureIDs`, i.e., the feature IDs resolved so far, DO need to be passed. That's because STSWebIdentity and SSO resolvers use `smithy.api#noAuth` operations & there will be no resolved identity to easily pass along this value like it's done with STSAssumeRole above.